neurots/extract_input/input_distributions.py

```diff
@@ -77,6 +79,11 @@ def distributions(
             neurite_type_warning(neurite_type)
             neurite_types[i] = neurite_type + "_dendrite"
 
+    if trunk_method not in ("simple", "3d_angles"):
```

Comment:
For this use case, there is little to no difference.

But in general when you want to check sth against a list of objects, it's better using a set, instead of a tuple/list because you can check inclusion in O(1) instead of O(n).